site/source/tutorials/005_form_widgets/demo/finished/biz-e-corp/src/widgets/Banner.ts

```diff
@@ -0,0 +1,15 @@
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+import { v } from '@dojo/widget-core/d';
+
+export default class Banner extends WidgetBase<WidgetProperties> {
+	render() {
+		return v('div', {}, [
+			v('h1', {
+				styles: {
+					'text-align': 'center'
```

Comment:
I think we definitely want to move away from inline styles as a suggestion since it really is an anti-pattern that we don't want to encourage early on, and there are more interesting attributes we can show. Happy for @rishson and/or @vansimke to give their thoughts though they are out today.